src/fallbacks.jl

```diff
@@ -39,7 +42,9 @@ nhole(t::AbstractPolygonTrait, geom) = nring(t, geom) - 1
 gethole(t::AbstractPolygonTrait, geom) = (getgeom(t, geom, i) for i in 2:ngeom(t, geom))
 gethole(t::AbstractPolygonTrait, geom, i) = getring(t, geom, i + 1)
 npoint(t::AbstractPolygonTrait, geom) = sum(npoint(p) for p in getring(t, geom))
-getpoint(t::AbstractPolygonTrait, geom) = flatten((p for p in getpoint(r)) for r in getring(t, geom))
+getpoint(t::PolygonTrait, geom) = flatten((p for p in getpoint(r)) for r in getring(t, geom))
+getpoint(t::AbstractPolygonTrait, geom) = getgeom(t, geom)
+getpoint(t::AbstractPolygonTrait, geom, i) = getgeom(t, geom, i)
```

Comment:
Is this correct? Won't this return rings? These definitions are the same as `getring` right above.

Reply:
The other `AbstractPolygonTrait` types besides `Polygon` don't have rings. Its a pretty bad a bug currently that we treat them like they do.

Reply:
I think they should behave as single rings? So a Triangle is a Polygon with a single ring, we should not skip directly to points.

Reply:
Its a bit confusing as in the diagrams on page 14 and 30 of the spec triangles are not built from a LinearRing: 

file:///home/raf/Downloads/06-103r4_Implementation_Specification_for_Geographic_Information_-_Simple_feature_access_-_Part_1_Common_Architecture_v1.2.1.pdf

Do you have a reference for where its specified that they have a single ring?

Reply:
Bump @evetion this is blocking. I don't think there are any implementations of `Trangle` anywhere yet either.

Is the idea that `getgeom` on a `TriangleTrait` and others will always return a length one iterator of `LinearRingTrait`?

I guess that can also make sense? `getpoint` will still return an iterator over the points so it doesn't make so much difference. One question is if wrappers will require the nesting in constructors - will a vector of `PointTrait` be acceptable input for a `Triangle` or does it have to be vector of vector of `PointTrait`?